R/attach.R

```diff
@@ -1,4 +1,14 @@
-core <- c("ggplot2", "tibble", "tidyr", "readr", "purrr", "dplyr", "stringr", "forcats", "lubridate")
+core <- c(
+  "ggplot2",
+  "tibble",
+  "tidyr",
+  "readr",
+  "purrr",
+  "dplyr",
+  "stringr",
+  "forcats",
+  "lubridate"
```

Comment:
```suggestion
  "lubridate",
  NULL
```

(just something I like to do with `c()` when possible)

Reply:
I think in principle that's a good idea, but it sort of implies that this list might grow, which I think is unlikely at this point in time.